honeycomb-core/src/attributes/manager.rs

```diff
@@ -381,9 +368,10 @@ impl AttrStorageManager {
         }
         .is_some()
         {
-            Err(ManagerError::DuplicateStorage)
-        } else {
-            Ok(())
+            panic!(
+                "E: Storage of attribute `{}` already exists in the attribute storage manager",
+                std::any::type_name::<A>()
+            );
```

Comment:
I am not a massive fan of `panic!` in a library.

You can leave it if you think the user cannot recover this error, but I think it is a regression there.

Reply:
I switched it to a warning in d86f664. That's a good point about panic in libraries; I'll try to go over all of those in a separate PR